src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -79,5 +79,5 @@ export const ATTRIBUTE_NAME_SOURCE = 'source';
 export const KEY_CODE_FOR_ENTER = 13;
 export const KEY_CODE_FOR_ESCAPE = 27;
 export const KEY_CODE_FOR_SPACE = 32;
-
+export const LEVELS_TO_CHECK = 2;
```

Comment:
When i look in this file, what does LEVELS_TO_CHECK mean or stand for?

Reply:
`LEVELS_TO_CHECK`is a constant that determines how deep the `recursivelyFetchConceptMappings` function should go when looking for mappings to a parent Concept. I have renamed it to `MAPPINGS_RECURSION_DEPTH`